kafka/producer/kafka.py

```diff
@@ -551,6 +549,9 @@ def send(self, topic, value=None, key=None, partition=None, timestamp_ms=None):
             value_bytes = self._serialize(
                 self.config['value_serializer'],
                 topic, value)
+            assert type(key_bytes) in (bytes, bytearray, memoryview, type(None))
+            assert type(value_bytes) in (bytes, bytearray, memoryview, type(None))
```

Comment:
Should this be using `isinstance` rather than `type()`? 

sorry for the late review--we had a new baby two weeks ago, so I'm still catching up on things...